### ISSUE TITLE
File widget aria-labels are missing when filtering

### ISSUE THREAD

**Issue by jacobtylerwalls** [archesproject/arches#9947]
Due to some duplicated code in the file widget, the accessibility features added in #9419 are missing if the list of files is filtered.

Found and fixed while dev'ing #9333.

**Comment by jacobtylerwalls** [comment]
Fixed in #9948
